website/docs/docs/dbt-versions/release-notes/04-Nov-2022/ide-features-ide-deprecation.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title: "Extra features in new IDE, and classic IDE deprecation"
+id: "ide-features-ide-deprecation"
```

Comment:
specificity could also be helpful here too (e.g., newer features and more deprecations)